app/enterprise/2.1.x/deployment/installation/ubuntu.md

```diff
@@ -68,17 +68,16 @@ You should now have two files in your home directory on the target system:
 
 ## Step 2. Install Kong Enterprise
 
-1. Update APT and install dependencies
+1. Update APT
 
     ```bash
     $ sudo apt-get update
-    $ sudo apt-get install openssl libpcre3 procps perl
     ```
 
 2. Install Kong Enterprise
 
     ```bash
-    $ sudo dpkg -i kong-enterprise-edition-<VERSION_NUMBER>.deb
+    $ sudo apt-get install $(realpath kong-enterprise-edition-{{site.data.kong_latest.version}}.*.deb)
```

Comment:
```suggestion
    $ sudo apt-get install $(realpath kong-enterprise-edition-{{page.kong_latest.version}}.*.deb)
```
I'd stick with this variable for the Enterprise docs - the site-level one pulls the latest version of Kong, whereas the page-level one pulls the version for the page - so it stays relevant in versioned docs. For the CE install docs, this isn't an issue, since they're not versioned.